src/main/resources/beam-template.conf

```diff
@@ -663,6 +663,8 @@ beam.debug {
 
     vmInformation.createGCClassHistogram = "boolean | false"
     writeModeChoiceAlternatives = "boolean | false"
+
+    writeRealizedModeChoiceFile = "boolean | false"
```

Comment:
Should there be anything added to the docs so that this isn't a hidden feature?

Reply:
I haven't found any mentions of "beam.debug" config in docs. Did I miss it somewhere? (https://github.com/LBNL-UCB-STI/beam/blob/develop/docs/inputs.rst] looks like a correct place. Shall we create a task to document the whole section?

Reply:
`beam-template.conf` is also a doc, comments may be added for details. Let's consider `writeRealizedModeChoiceFile ` naming descriptive enough.